### PR TITLE
[MWRAPPER-103] Apply recommended workaround

### PR DESCRIPTION
**Pull request by cstamas**
As Marcono1234 on JIRA issue recommended, added workaround just before the use of `Get-FileHash` call.

---

https://issues.apache.org/jira/browse/MWRAPPER-103

### CODE REVIEW

File: maven-wrapper-distribution/src/resources/mvnw.cmd [hunk 1]
```diff
@@ -160,6 +160,7 @@ FOR /F "usebackq tokens=1,2 delims==" %%A IN ("%MAVEN_PROJECTBASEDIR%\.mvn\wrapp
 )
 IF NOT %WRAPPER_SHA_256_SUM%=="" (
     powershell -Command "&{"^
+       "Import-Module $PSHOME\Modules\Microsoft.PowerShell.Utility -Function Get-FileHash"^
```
Comment (jorsol) [inline maven-wrapper-distribution/src/resources/mvnw.cmd:163]: I assume that a `;` is needed at the end of Get-FileHash.

Comment (cstamas) [reply maven-wrapper-distribution/src/resources/mvnw.cmd:163]: Hm, no idea, but Google found me this: https://poshcode.gitbook.io/powershell-practice-and-style/style-guide/code-layout-and-formatting#avoid-using-semicolons-as-line-terminators

Comment (cstamas) [reply maven-wrapper-distribution/src/resources/mvnw.cmd:163]: even if it is to avoid, will add to be consistent with existing PS code....

**Review by michael-o — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #292

**Pull request merged**